template/content/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fetchConfig = require('zero-config');
+
+var ApplicationClients = require('./clients.js');
+var Endpoints = require('./endpoints.js');
+
+module.exports = Application;
+
+function Application(options) {
+    if (!(this instanceof Application)) {
+        return new Application(options);
+    }
+
+    var self = this;
+    options = options || {};
+
+    self.config = fetchConfig(__dirname, {
+        dcValue: 'todo',
+        seed: options.seedConfig,
+        loose: false
+    });
+
+    self.clients = ApplicationClients(self.config, {
+        logger: options.logger,
+        statsd: options.statsd
+    });
+    self.endpoints = Endpoints(self);
+
+    // Example data structure on application
+    self.exampleDb = {};
```

Comment:
Im in favor of getting rid of `exampleDb`. If this generator is the scaffolding meant to be built on top of, every developer is going to have to remove this line of code from their app. I think that's wasted time. And this `exampleDb` is best left for an "example app".